examples/DotNetCore/WebDemoWithEventHub/WebDemoWithEventHub/Program.cs

```diff
@@ -42,8 +42,11 @@ public static void Main(string[] args)
                                        .Select(keyFilter: "WebDemo:*")
                                        .ConfigureRefresh((refreshOptions) =>
                                        {
-                                           // Indicates that all configuration should be refreshed when the given key has changed.
-                                           refreshOptions.Register(key: "WebDemo:Sentinel", refreshAll: true);
+                                           refreshOptions.Register(key: "WebDemo:*", refreshAll: true);
```

Comment:
This won't work. We don't support watching on *. Why is the change?

Reply:
I see. But the IConfigurationRoot's Refresh method still did refresh the entire configuration. If we don't have a regex matching in the provider, then I'm not sure if this line is doing anything.

---
In reply to: [432185410](https://github.com/Azure/AppConfiguration/pull/329#discussion_r432185410) [](ancestors = 432185410)

Reply:
`IConfigurationRoot.Reload` doesn't use the settings specified for refresh using `ConfigureRefresh`. In case we decide to continue with this approach, then users don't need to register specific settings for refresh in their code. For other alternatives that don't use `Reload`, these settings might be needed.